### DISCUSSION TITLE
Where are content titles stored in the DB?

### DISCUSSION THREAD

**Discussion by ametsoc45** [MasaCMS/MasaCMS#308] [Q&A]
I used to know this, but a simple peek at tcontent had me down the rabbit hole. Could someone remind me where Page Title, Navigation Title, URL Title, HTML Browser Title are stored in the DB?

Thanks!

**Comment by Tigerlady** [comment]
Brian,

They are in the tcontent table highlighted in green below.

tcontent_ID    int    Unchecked
SiteID    nvarchar(25)    Checked
ModuleID    char(35)    Checked
ParentID    char(35)    Checked
ContentID    char(35)    Checked
ContentHistID    char(35)    Checked
RemoteID    nvarchar(255)    Checked
RemoteURL    nvarchar(255)    Checked
RemotePubDate    nvarchar(50)    Checked
RemoteSourceURL    nvarchar(255)    Checked
RemoteSource    nvarchar(255)    Checked
Credits    nvarchar(255)    Checked
FileID    char(35)    Checked
Template    nvarchar(50)    Checked
Type    nvarchar(25)    Checked
subType    nvarchar(25)    Checked
Active    tinyint    Checked
OrderNo    int    Checked
Title    nvarchar(255)    Checked
MenuTitle    nvarchar(255)    Checked
Summary    nvarchar(MAX)    Checked
Filename    nvarchar(255)    Checked
MetaDesc    nvarchar(MAX)    Checked
MetaKeyWords    nvarchar(MAX)    Checked
Body    nvarchar(MAX)    Checked
lastUpdate    datetime    Checked
lastUpdateBy    nvarchar(50)    Checked
lastUpdateByID    nvarchar(50)    Checked
DisplayStart    datetime    Checked
DisplayStop    datetime    Checked
Display    tinyint    Checked
Approved    tinyint    Checked
IsNav    tinyint    Checked
Restricted    tinyint    Checked
RestrictGroups    nvarchar(255)    Checked
Target    nvarchar(50)    Checked
TargetParams    nvarchar(255)    Checked
responseChart    tinyint    Checked
responseMessage    nvarchar(MAX)    Checked
responseSendTo    nvarchar(MAX)    Checked
responseDisplayFields    nvarchar(MAX)    Checked
moduleAssign    nvarchar(255)    Checked
displayTitle    tinyint    Checked
Notes    nvarchar(MAX)    Checked
inheritObjects    nvarchar(25)    Checked
isFeature    tinyint    Checked
ReleaseDate    datetime    Checked
IsLocked    tinyint    Checked
nextN    int    Checked
sortBy    nvarchar(50)    Checked
sortDirection    nvarchar(50)    Checked
featureStart    datetime    Checked
featureStop    datetime    Checked
forceSSL    tinyint    Unchecked
audience    nvarchar(255)    Checked
keyPoints    nvarchar(MAX)    Checked
searchExclude    tinyint    Checked
path    nvarchar(MAX)    Checked
tags    nvarchar(MAX)    Checked
doCache    tinyint    Checked
urltitle    nvarchar(255)    Checked
htmltitle    nvarchar(255)    Checked
created    datetime    Checked
mobileExclude    tinyint    Checked
changesetID    char(35)    Checked
imageSize    nvarchar(50)    Checked
imageHeight    nvarchar(15)    Checked
imageWidth    nvarchar(15)    Checked
childTemplate    nvarchar(50)    Checked
majorVersion    int    Checked
minorVersion    int    Checked
expires    datetime    Checked
displayInterval    nvarchar(200)    Checked
objectParams    nvarchar(MAX)    Checked


Barbara Renowden
CTO/Co-Founder
Centric Web, Inc.
http://www.centricweb.com
630-541-9956
centricweb


------ Original Message ------
From "BrianM" ***@***.***>
To "MasaCMS/MasaCMS" ***@***.***>
Cc "Subscribed" ***@***.***>
Date 5/14/24, 4:46:28 PM
Subject [MasaCMS/MasaCMS] Where are content titles stored in the DB? 
(Discussion #308)


I used to know this, but a simple peek at tcontent had me down the 
rabbit hole. Could someone remind me where Page Title, Navigation Title, 
URL Title, HTML Browser Title are stored in the DB?

Thanks!

—
Reply to this email directly, view it on GitHub 
<https://github.com/MasaCMS/MasaCMS/discussions/308>, or unsubscribe 
<https://github.com/notifications/unsubscribe-auth/AAODXSLWOCQ442M453A5B6TZCKA3JAVCNFSM6AAAAABHW7AWKGVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGY3DQOJRGU>.
You are receiving this because you are subscribed to this thread.Message 
ID: ***@***.***>
